src/SDKs/KeyVault/Management/Management.KeyVault/Generated/Models/StoragePermissions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since generated code has changes, there should be an update to generate.cmd (see the comment on the other generate.cmd)